src/CommunicationsMicroservice/Communications.Api/Program.cs

```diff
@@ -1,25 +1,32 @@
+using Communications.Api.Configuration;
+using Communications.Api.Middlewares;
+
 var builder = WebApplication.CreateBuilder(args);
+var services = builder.Services;
+var configuration = builder.Configuration;
 
-// Add services to the container.
+services.ConfigureInfrastructure(configuration);
+services.ConfigureUtilities();
 
-builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+services.AddControllers();
+services.AddEndpointsApiExplorer();
+services.AddSwaggerGen();
 
 var app = builder.Build();
 
-// Configure the HTTP request pipeline.
 if (app.Environment.IsDevelopment())
```

Comment:
Could you explain this condition?

Reply:
If the app's environment variable is Development this condition returns true. This means the code within curly braces will be executed.